architecture/routing.adoc

```diff
@@ -61,19 +60,19 @@ Example secure route (using edge termination)
 }
 ----
 
+====
+
+Note that while both secure and unsecure is possible with an OpenShift deployment, Red Hat recommends a secure routing service.
+
 == Securing Routes
-Creating a secure route to your pods can be accomplished by specifying the TLS Termination of the route and, optionally,
-providing certificates to use. As of writing, OpenShift beta1 TLS termination relies on SNI for serving custom certificates (in the reference HAProxy implementation).
-Any non-SNI traffic received on port 443 will have TLS terminated with a generic certificate.  In the future, the ability
-to create custom frontends within the router will allow all traffic to serve custom certificates.
+Creating a secure route to your pods is accomplished by specifying the TLS Termination of the route and, optionally, providing certificates. At the time of writing, OpenShift Beta 1 TLS termination relies on SNI for serving custom certificates. Any non-SNI traffic received on port 443 has TLS termination with a generic certificate. In the future, the ability to create custom frontends within the router will allow all traffic to serve custom certificates.
 
-There are three types of TLS termination:
+By default, OpenShift has no pod encryption, but can be set to any of the following three types of TLS termination:
```

Comment:
I would remove the reference to pod here, we're trying to keep from directly tying a route to a pod.  Perhaps, "by default, OpenShift routes are unsecure"